model/riscv_fdext_regs.sail

```diff
@@ -103,18 +103,18 @@ register f30 : fregtype
 register f31 : fregtype
 
 function dirty_fd_context() -> unit = {
-  assert(sys_enable_fdext());
+  assert(hartSupports(Ext_F));
   mstatus[FS] = extStatus_to_bits(Dirty);
   mstatus[SD] = 0b1;
 }
 
 function dirty_fd_context_if_present() -> unit = {
-  assert(sys_enable_fdext() != sys_enable_zfinx());
-  if sys_enable_fdext() then dirty_fd_context()
+  assert(hartSupports(Ext_F) != hartSupports(Ext_Zfinx));
+  if hartSupports(Ext_F) then dirty_fd_context()
 }
 
 function rF (Fregno(r) : fregno) -> flenbits = {
-  assert(sys_enable_fdext());
+  assert(hartSupports(Ext_F));
```

Comment:
Again, perhaps this should be 
```suggestion
  assert(currentlyEnabled(Ext_F));
```
here and in the various read/write accessors below.